icicle/src/vec_ops/vec_ops.cu

```diff
@@ -164,4 +189,66 @@ namespace vec_ops {
 
     return CHK_LAST();
   }
+
+  template <typename E>
```

Comment:
existing template for vec_ops cannot be reused here?

Reply:
I deleted the inplace template, it's merged with `bit_reverse`